go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -119,6 +119,15 @@ func tryCreateTxThrottler(env tabletenv.Env, topoServer *topo.Server) (*TxThrott
 		return newTxThrottler(env, &txThrottlerConfig{enabled: false})
 	}
 
+	for _, tabletType := range env.Config().TxThrottlerTabletTypes {
+		switch tabletType {
+		case topodatapb.TabletType_REPLICA, topodatapb.TabletType_RDONLY:
+			continue
+		default:
+			return nil, fmt.Errorf("%q tablet type is not supported", tabletType)
```

Comment:
Please use `vterrors.Errorf( .... )` and supply an appropriate error code.
vterrors are more formal, are used to wrap and unwrap other errors, and are easier to detect in logs.